### PR TITLE
Support to package tar

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

**Review by yixinglu — DISMISSED** [review]

**Review by jievince — DISMISSED** [review]

**Comment by yixinglu** [comment]
Please solve the conflicts

**Review by yixinglu — COMMENTED** [review]
Does this install script support ARM arch platform?

File: package/package.sh [hunk 1]
```diff
@@ -206,7 +233,54 @@ function package {
...
-build $version $enablesanitizer $static_sanitizer $build_type $branch
-package $strip_enable
+gen_package_name
+
+if [[ $package_type == "auto" ]]; then
+    build
+    package
+else
+    install_dir=${build_dir}/install
+    build $version $enablesanitizer $static_sanitizer $buildtype
```
Comment (yixinglu) [inline package/package.sh:277]: these arguments could be removed and need not to process the inputs in the `build` function.

**Comment by CPWstatic** [comment]
close #269

**Comment by Shylock-Hg** [comment]
Close https://github.com/vesoft-inc/nebula/issues/2750

**Pull request closed without merging**